src/components/FilterStore/FilterStore.jsx

```diff
@@ -52,9 +52,9 @@ const FilterStore = (props) => {
         <FormControlLabel
           control={
             <Checkbox
-              checked={state.checkedD}
+              checked={state.cuarzos}
```

Comment:
these names are currently hardcoded, this categories must came from backend as soon backend supports the category endpoint, for know this is a ToDo task, the hardcoded categories is ok now